modules/docs/sc_event.md

```diff
@@ -697,7 +699,13 @@ local result = test_event:is_valid_bv()
 
 ## find_hostgroup_in_list method
 
-The **find_hostgroup_in_list** method checks if one of the hostgroup in [**accepted_hostgroups**](sc_param.md#default-parameters) is linked to the host.
+The **find_hostgroup_in_list** method checks if one of the hostgroup in the hostgroups list parameter ([**accepted_hostgroups or rejected_hostgroups parameters**](sc_param.md#default-parameters)) is linked to the host.
+
+### find_hostgroup_in_list: parameters
+
+| parameter            | type   | optional | default value |
+| -------------------- | ------ | -------- | ------------- |
+| a list of hostgroups | string | no       |               |
```

Comment:
```suggestion
| a coma separated list of hostgroups | string | no       |               |
```

Reply:
don't mind the fucked up indentation, it'll be fixed when we update the doc from code editor